examples/tfengine/generated/org_foundation/audit/main.tf

```diff
@@ -95,9 +95,9 @@ module "storage_export" {
   source  = "terraform-google-modules/log-export/google"
   version = "~> 6.0.0"
 
-  log_sink_name          = "storage-audit-logs-sink"
+  log_sink_name          = var.logs_storage_bucket.sink_name
   destination_uri        = module.storage_destination.destination_uri
-  filter                 = "logName:\"logs/cloudaudit.googleapis.com\""
+  filter                 = join(" OR ", concat(["logName:\"logs/cloudaudit.googleapis.com\""], var.additional_filters))
```

Comment:
have you tried deployed and tested it?

Reply:
Not yet. I assumed tests cared about that. Let me try it.

Is there any particular recommendation on what I'm searching for? I'm not sure if everything can be deployed correctly with a malformed filter string without logged errors.

Reply:
The integration tests don't deploy the audit recipe i believe. The unit tests might run a local terraform plan on the audit recipe. Can you try deploy the audit recipe in our testing org and pass in some additional filter strings, then verify if the filter is set properly by referring to log export doc here https://cloud.google.com/logging/docs/export/configure_export_v2

Reply:
I deployed an audit example and created these two sinks:

![Screenshot 2021-07-15 1 05 08 PM](https://user-images.githubusercontent.com/33379285/125836061-7e50fac7-293e-405a-b45c-6fe1675f0132.png)

filters: `logName:"logs/cloudaudit.googleapis.com" OR logName="logs/forseti" OR logName="logs/application"`

![Screenshot 2021-07-15 1 04 44 PM](https://user-images.githubusercontent.com/33379285/125836062-55dbc616-91cf-4e90-87bb-682be9111ff6.png)

filters: `logName:"logs/cloudaudit.googleapis.com" OR logName="logs/forseti" OR logName="logs/application"`

Reply:
So far, everything seems to be ok. I'll leave the config for a few hours so you can check it out before destroying it.